### PR TITLE
Please pull newly created x-common/doc/

### PR DESCRIPTION
**Pull request by studiolo**
This fixes https://github.com/exercism/rossconf/issues/10

It has task-oriented directories now:
doc/          # General documentation
  installing/   # Everything you need to get up and running.
  coding/       # Programming the solutions to the exercises.
  discussing/   # Giving and receiving feedback.
  contributing/ # Participating in the project.

### CODE REVIEW

File: doc/contributing/contributing.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+https://github.com/exercism/configlet#configlet
+
+You can download the latest release from the releases page in the [configlet
+repo](https://github.com/exercism/configlet/releases), or you can use the
+`bin/fetch-configlet` command from the root of the language track repository,
+which will make a guess at what operating system and architecture you have and
+attempt to download the right one.
+
+Then, verify the config by calling `bin/configlet .` (notice the dot). This says
+_check the config of the language track that is stored right here).
```
Comment (kytrinyx) [inline doc/contributing/contributing.md]: We're missing an underscore here (probably a typo in the original), and also there is an extra parenthesis. It should be:

_check the config of the language track that is stored right here_.

File: doc/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+In a way, the entire site happened by accident as Exercism spread by word of mouth, whereby current users helped new users get started. If you happened to come to the site without the support of someone more experienced, it was very difficult to get started.
+
+If you’re having trouble getting started, you’ll get the fastest answers [on GitHub](https://github.com/exercism/exercism.io/issues). The best place to start is to look at current issues. If you don’t see your issue listed there, please [submit a new issue](https://github.com/exercism/exercism.io/issues/new).
+
+You can also visit our [support chat](https://gitter.im/exercism/support). We're justgetting started with this, so activity is quite low, but the more people chat there, the more people will hang out.
+
+Thank you for your patience as we improve the site. We’re so happy to have you here.
+
+### Exercises</h2>
```
Comment (kytrinyx) [inline doc/getting-started.md]: There's a stray `h2` tag here.

Comment (kytrinyx) [inline doc/getting-started.md]: > `justgetting`

`just getting`

**Comment by studiolo** [comment]
fixed small md formatting errors

**Comment by kytrinyx** [comment]
Thanks so much for working on this! It's going to be so great to get the documentation into better shape.

**Pull request merged**